strapi/strapi
# fix: Add dependency description for development environment.

### What does it do?

Add dependency descriptions for development environments.

### Why is it needed?

When I was building the build development environment (yarn setup) on my mac system, I found that it was executing with errors. After troubleshooting, I found that the reason is that my python version is python3, and the build environment dependency is python2. 
So I think I need to add a note about it in the documentation to avoid other contributors encountering similar problems.

### How to test it?

In a python3 environment, clone the project and execute yarn setup, and you will see the error. After changing the python version to python2, everything works fine.
